packages/marketplace/src/components/ui/unsupport-brower-pop-up.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react'
+import { ToastMessage } from '@reapit/elements'
+import styles from '@/styles/blocks/sandbox-pop-up.scss?mod'
+import { useSelector } from 'react-redux'
+import { selectRefreshSession } from '@/selector/auth'
+
+export const TIMEOUT_DURATION = 500
+
+export const popUp = (setOpen, unsupported) => () => {
+  if (unsupported) {
+    const timeout = setTimeout(setOpen, TIMEOUT_DURATION)
+    return () => {
+      clearTimeout(timeout)
+    }
+  }
+}
+
+export const UnsupportBrowerPopUp = ({
+  unsupported = false,
+  // eslint-disable-next-line
+  message = 'Unsupported Browser - Unfortunately as Internet Explorer 11 is no longer supported, we are unable to display the documentation, please login to the Developer Portal using either Chrome, Firefox, Safari or Edge',
+}) => {
+  const refreshSession = useSelector(selectRefreshSession)
+  const loginMode = refreshSession?.mode
+
+  const [isOpen, setIsOpen] = React.useState<boolean>(false)
+  const setOpen = React.useCallback(setIsOpen.bind(null, true), [setIsOpen])
```

Comment:
Do we really need to use bind() function here? It's hard to extend in the future when we need to do something more than just call `setIsOpen` function. By the way, `setIsOpen` reference won't be changed after every renders so it's unnecessary to put that function into `useCallback `dependency array